### PR TITLE
Update Network Edge Security Service sweeper

### PR DESCRIPTION
**Pull request by shuyama1**
The resources are created in several regions. Currently handwrite the sweeper to support it in multi regions. Created ticket https://github.com/hashicorp/terraform-provider-google/issues/17835 to track the support of it in mmv1 generator
 
Closes https://github.com/hashicorp/terraform-provider-google/issues/16745

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/third_party/terraform/services/compute/resource_compute_network_edge_security_service_sweeper.go.erb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+			})
+			if err != nil {
+				log.Printf("[INFO][SWEEPER_LOG] Error deleting for url %s : %s", deleteUrl, err)
+			} else {
+				log.Printf("[INFO][SWEEPER_LOG] Sent delete request for %s resource: %s", resourceName, name)
+			}
+		}
+
+		if nonPrefixCount > 0 {
+			log.Printf("[INFO][SWEEPER_LOG] %d items were non-sweepable and skipped.", nonPrefixCount)
```
Comment (zli82016) [inline mmv1/third_party/terraform/services/compute/resource_compute_network_edge_security_service_sweeper.go.erb:122]:
```suggestion
			log.Printf("[INFO][SWEEPER_LOG] %d items in %s were non-sweepable and skipped.", r, nonPrefixCount)
```

File: mmv1/third_party/terraform/services/compute/resource_compute_network_edge_security_service_sweeper.go.erb [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+		resourceList, ok := res["networkEdgeSecurityServices"]
+		if !ok {
+			log.Printf("[INFO][SWEEPER_LOG] Nothing found in response.")
+			return nil
+		}
+
+		rl := resourceList.([]interface{})
+
+		log.Printf("[INFO][SWEEPER_LOG] Found %d items in %s list response.", len(rl), resourceName)
```
Comment (zli82016) [inline mmv1/third_party/terraform/services/compute/resource_compute_network_edge_security_service_sweeper.go.erb:81]:
```suggestion
		log.Printf("[INFO][SWEEPER_LOG] Found %d items in %s in %s list response.", len(rl), r, resourceName)
```

File: mmv1/third_party/terraform/services/compute/resource_compute_network_edge_security_service_sweeper.go.erb [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+		log.Printf("[INFO][SWEEPER_LOG] error loading: %s", err)
+		return err
+	}
+
+	t := &testing.T{}
+	billingId := envvar.GetTestBillingAccountFromEnv(t)
+
+	regions := []string{"us-central1", "us-west2", "us-south1", "southamerica-west1", "europe-west1"}
+	for _, r := range regions {
+		// Setup variables to replace in list template
```
Comment (zli82016) [inline mmv1/third_party/terraform/services/compute/resource_compute_network_edge_security_service_sweeper.go.erb:43]: We can add a log here like `log.Printf("[INFO][SWEEPER_LOG] Starting sweeper for %s in %s", resourceName, r)
` ?

**Comment by zli82016** [comment]
LGTM mostly. It is better to add the region to the logs?

**Comment by shuyama1** [comment]
> LGTM mostly. It is better to add the region to the logs?

Added! Thanks for the suggestion!

**Review by zli82016 — APPROVED** [review]
LGTM

**Pull request merged**